task16/css/style.css

```diff
@@ -0,0 +1,154 @@
+body {
+  height: 100vh;
+  padding: 60px;
+  background-color: rgb(234, 235, 236);
+  font-family: Arial, Helvetica, sans-serif;
+  letter-spacing: 0.1em;
+}
+
+.loader,
+.loader:before,
+.loader:after {
+  background: #ffffff;
+  -webkit-animation: load1 1s infinite ease-in-out;
+  animation: load1 1s infinite ease-in-out;
+  width: 1em;
+  height: 4em;
+}
+.loader {
+  color: #ffffff;
+  text-indent: -9999em;
+  margin: 88px auto;
+  position: relative;
+  font-size: 11px;
+  -webkit-transform: translateZ(0);
+  -ms-transform: translateZ(0);
+  transform: translateZ(0);
+  -webkit-animation-delay: -0.16s;
+  animation-delay: -0.16s;
+}
+.loader:before,
+.loader:after {
+  position: absolute;
+  top: 0;
+  content: "";
+}
+.loader:before {
+  left: -1.5em;
+  -webkit-animation-delay: -0.32s;
+  animation-delay: -0.32s;
+}
+.loader:after {
+  left: 1.5em;
+}
+@-webkit-keyframes load1 {
+  0%,
+  80%,
+  100% {
+    box-shadow: 0 0;
+    height: 4em;
+  }
+  40% {
+    box-shadow: 0 -2em;
+    height: 5em;
+  }
+}
+@keyframes load1 {
+  0%,
+  80%,
+  100% {
+    box-shadow: 0 0;
+    height: 4em;
+  }
+  40% {
+    box-shadow: 0 -2em;
+    height: 5em;
+  }
+}
+
+.tab-menu {
+  max-width: 760px;
+  margin: 0 auto;
+  overflow: hidden;
+  display: flex;
+  justify-content: space-between;
+}
+
+.menu-list {
+  padding: 15px 0 15px;
+  width: 188px;
+  text-align: center;
+  color: #fff;
+  background-color: rgb(8, 2, 58);
+  border-radius: 10px 10px 0 0;
+  font-weight: bold;
+  font-size: 15px;
+  text-decoration: none;
+}
```

Comment:
NITS & IMO
I think it is common to make the cursor a pointer where it can be pressed.
(The specification does not include CSS, so I'll leave it to sen-san.)
```suggestion
.menu-list {
  padding: 15px 0 15px;
  width: 188px;
  text-align: center;
  color: #fff;
  background-color: rgb(8, 2, 58);
  border-radius: 10px 10px 0 0;
  font-weight: bold;
  font-size: 15px;
  text-decoration: none;
  cursor: pointer;
}
```

Reply:
I would not have noticed it if you had not pointed it out. Thank you very much.
f7aaf72